vendor/github.com/go-openapi/errors/headers.go

```diff
@@ -74,3 +74,12 @@ func InvalidResponseFormat(value string, allowed []string) *Validation {
 		message: fmt.Sprintf(responseFormatFail, allowed),
 	}
 }
+
+// Validate error message name for aliased property
+func (e *Validation) ValidateName(name string) *Validation {
```

Comment:
I think the above change in `vendor/github.com/go-openapi/errors/headers.go` still need to be kept, right?